packages/flash_db/src/flash_db/models.py

```
@@ -1,6 +1,6 @@
 from __future__ import annotations
 
-from datetime import datetime  # noqa: TC003
+from datetime import datetime  # noqa: TCH003
```

Ruff (0.14.14)

Unused `noqa` directive (non-enabled: `TCH003`)

Remove unused `noqa` directive

(RUF100)